DESCRIPTION

```diff
@@ -54,7 +54,7 @@ Imports:
     methods,
     Biobase,
     BiocGenerics,
-    ProtGenerics (>= 1.35.4),
+    ProtGenerics (>= 1.37.1),
```

Comment:
Is that bump required, or is that simply the tested version ?

Reply:
Exactly. The `estimatePrecursorIntensity` generic was added to *ProtGenerics* version 1.37.1